scripts/build_clang_debug.sh

```diff
@@ -7,7 +7,7 @@ BUILD_DIR=${BUILD_PREFIX}build_clang_debug
 mkdir -p ${BUILD_DIR}
 cd ${BUILD_DIR}
 cmake \
-    -DCMAKE_CXX_FLAGS='-std=c++17 -Wall -Wextra -pedantic -Werror' \
+    -DCMAKE_CXX_FLAGS='-std=c++1z -Wall -Wextra -pedantic -Werror' \
```

Comment:
Why `1z`?

Reply:
Mac OS stock compiler has no such option like C++17. So I can revert this change.

Reply:
Too bad for mac. Revert, please.

Reply:
Well, if we want to be user-friendly we must support it - nobody needs a product to compile it only for docker image ;) I'll fix it a little bit later.

Reply:
**Done**